bedrock/firefox/firefox_details.py

```diff
@@ -107,93 +123,148 @@ def _get_filtered_builds(self, builds, version, query=None):
             if query is not None and not self._matches_query(build_info, query):
                 continue
 
-            for plat in platforms:
-                build_info['platforms'][plat] = {
-                    'download_url': self.get_download_url(plat, locale,
-                                                          version),
-                }
-
-            # Append a Linux 64-bit build
-            if 'Linux' in platforms:
-                build_info['platforms']['Linux 64'] = {
-                    'download_url': self.get_download_url('Linux 64', locale,
-                                                          version),
+            for platform in self.platform_info.keys():
```

Comment:
No, this works. Currently all locales have builds for all platforms, so it just iterates 5 platform targets instead of `platforms`.